packages/kbn-es/src/install/archive.js

```diff
@@ -69,7 +71,13 @@ exports.installArchive = async function installArchive(archive, options = {}) {
     await appendToConfig(installPath, 'xpack.security.enabled', 'true');
 
     await appendToConfig(installPath, 'xpack.license.self_generated.type', license);
-    await configureKeystore(installPath, password, log, bundledJDK);
+    await configureKeystore(
+      installPath,
+      password,
```

Comment:
__note:__ decided to keep `password` separate from `secureSettings` even though the code that adds secure settings could be reused for password as well. What do you think?

Reply:
I actually really like the idea of combing them and just manually passing `[['bootstrap.plassword', password], ...parseSettings(esArgs, { filter: SettingsFilter.SecureOnly })]` to `configureKeystore`

Reply:
You got it!